src/components/bulk-email-tool/bulk-email-form/BulkEmailForm.jsx

```diff
@@ -167,6 +160,17 @@ export default function BulkEmailForm(props) {
             </Form.Control.Feedback>
           )}
         </Form.Group>
+        <div>
+          <p>
+            We recommend sending learners no more than one email message per week. Before you send your email, review
+            the text carefully and send it to yourself first, so that you can preview the formatting and make sure
+            embedded images and links work correctly.
+          </p>
+          <p>
+            <strong>CAUTION!</strong> When you select Send Email, your email message is added to the queue for sending,
+            and cannot be cancelled.
+          </p>
+        </div>
```

Comment:
We should put this in a <FormattedMessage> to support i18n.

Reply:
Oh you think I would learn this lesson by now but here we are.